lib/obj.js

```diff
@@ -6,6 +6,7 @@
 
 /*
  * Copyright 2020 Joyent, Inc.
+ * Copyright 2025 Edgecast Cloud LLC.
```

Comment:
Same copyright question as in TritonDataCenter/node-libmanta#16 for @goekesmi .

Reply:
I just updated this branch with the copyright under your name. I think is easier that we abandon this PR and you use this branch to submit a new one, so only your name appears in this PR. The reason is that I still see my name and I already changed the author.